sorc/build_aigefs_ensstat.sh

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env bash
+set -eux
+
+module reset
+cwd=`pwd`
+
+progname=aigefs_ensstat
+
+source ../versions/build.ver
+
+module use ${cwd}/../modulefiles/aigefs
+module load ${progname}
+module list
+
+# Check final exec folder exists
+if [ ! -d "../exec" ]; then
+  mkdir ../exec
+fi
+
+#
+cd ${progname}.fd
+
+export FCMP=${FCMP:-ftn}
+export FCMP95=$FCMP
+
+if [ "$DEBUG" = 'Y' ] ; then
+  #export FFLAGSM="-O0 -traceback -check all -ftrapuv -convert big_endian"
+  export FFLAGSM="-O0 -traceback -g -check all -ftrapuv -convert big_endian"
+else
+  export FFLAGSM="-O3 -traceback -convert big_endian"
```

Comment:
```suggestion
  export FFLAGSM="-O3 -g -traceback -convert big_endian"
```
-g should be retained